### ISSUE TITLE
Build process

### ISSUE THREAD

**Issue by grantmcdermott** [stata2r/stata2r.github.io#5]
@kylebutts Pending small changes like reverting back to two-column format, the site looks great. But I'm a bit worried about the new build (really: writing) process. The key `commands.txt` file is kinda hard to navigate and doesn't give much in the way of visual feedback. I just made some minor changes and honestly had to iterate through `commands.txt` -> `create_pages.R` a few times to catch simple things I'd missed in the raw text environment.

For ease of writing and immediate visual feedback, I'm wondering whether we shouldn't just use GitHub pages + .Rmd? We could always import your HTML theme, set `eval=FALSE` globally and use two-column formatting for the Stata-R code comparisons. (Something like [this](http://grantmcdermott.com/example/lecture/lecture.html).) 

Feel free to weigh-in too @NickCH-K.

**Comment by NickCH-K** [comment]
I haven't tried to use the new format yet so I don't know about how difficult it is to use. RMD makes sense but also I'm guessing this isn't going to require regular updates, so it may be lower effort to go through a tougher update process for a few iterations rather than changing it.

**Comment by grantmcdermott** [comment]
Yeah, honestly I was thinking something as simple as bundling the site up as a "fake" `pkgdown` site, where the **data.table** and **fixest** pages are really just vignettes. The [new bootsrap 5 support](https://pkgdown.r-lib.org/dev/index.html) makes theming easy and it also bundles cool features like a search bar. Again, not saying we _should_ switch after the work Kyle's put in. But if it's easy enough to port the good, whilst leveraging the scaffolding provided by something like `pkgdown` then it's worth considering.

**Comment by kylebutts** [comment]
I edited commands.txt to make it much easier to edit. Each example should take up 6 lines: Page, cat_id (for sidebar), Category, Example, stata_code, r_code. It's not necessary to separate the examples by blank lines 
(completely blank new lines aren't read by `scan()`), but it's easier visually. 

https://github.com/stata2r/stata2r.github.io/blob/f900c3e6f9f5c7bcbf8f2063fc01ed6fa53c6928/commands.txt#L1-L13

I think that should be easier to work with @grantmcdermott?

**Comment by grantmcdermott** [comment]
Thanks, yes that is better. (I had to change a few small things for the build to work, but I think those were just variable typos.)

I'll close this now, but just to let you both know that I've reworked quite a bit of the data section, so it might be worth pulling before making any changes on your sides. Also, I've spotted quite a lot of typos (missing commas, spaces, closing parens, etc) so we'll need to be careful about that going forward. Again, it's a little hard b/c everything is written on a single line with lots of `\n`s but ymmv.
